cmd/pg/copy.go

```diff
@@ -57,11 +57,11 @@ func init() {
 	backupCopyCmd.Flags().StringVarP(&backupName, backupNameFlag, backupNameShorthand, "", backupNameDescription)
 	backupCopyCmd.Flags().StringVarP(&toConfigFile, toFlag, toShorthand, "", toDescription)
 	backupCopyCmd.Flags().StringVarP(&fromConfigFile, fromFlag, fromShorthand, "", fromDescription)
-	backupCopyCmd.Flags().BoolVarP(&withoutHistory,
-		withoutHistoryFlag,
-		withoutHistoryShorthand,
+	backupCopyCmd.Flags().BoolVarP(&withAllHistory,
+		withAllHistoryFlag,
+		withAllHistoryShorthand,
 		false,
-		withoutHistoryDescription)
+		withAllHistoryDescription)
```

Comment:
it is backward incompatible changes... with changed defaults.
Is there real reason to do so?

Reply:
Backup without any WALs will not work anyway. So there is no chance that someone has used this code before (and then was able to restore the backup).

Reply:
In addition, there was an error without the "without history" flag - the function copied only from finish_lsn. Therefore, backups from walg copy were useless